packages/block-library/src/section/theme.scss

```diff
@@ -0,0 +1,126 @@
+$block-name: "section";
+
+.wp-block-#{$block-name} {
+	padding: $block-padding;
+
+	&.has-background {
+		margin-top: 0;
+		margin-bottom: 0;
+	}
+}
+
+/**
+ * All Alignment Settings
+ */
+.wp-block[data-type="core/#{$block-name}"] {
+
+	// Child: Wide Alignment
+	.wp-block[data-align="wide"] {
+		@include break-small() {
+			padding-left: $block-padding*2;
+			padding-right: $block-padding*2;
+		}
+	}
+
+	// Child: Full Alignment
+	.editor-block-list__layout > .wp-block[data-align="full"] {
+		left: auto;
+		max-width: 100%;
+		margin-left: 0;
+		margin-right: 0;
+		padding-left: 0;
+		padding-right: 0;
+
+		.editor-block-list__block-edit {
+			margin-left: 0;
+			margin-right: 0;
+		}
+	}
+}
+
+
+/**
+ * Section Wide Alignment
+ */
+.wp-block[data-type="core/#{$block-name}"][data-align="wide"] {
+
+	// Child: Standard Alignment
+	.wp-block:not([data-align="wide"]):not([data-align="full"]) {
+		@include break-wide() {
+			// width: calc((8 * (100vw / 12 ) ) - #{$block-padding});
+		}
+
+		@include break-xlarge() {
+			width: 100%;
+			// max-width: calc((6 * (100vw / 12 ) ) - #{$block-padding});
+		}
+	}
+
+	// Wide
+	.wp-block[data-align="wide"] {
+		padding-left: $block-padding*2 + 2px;
+		padding-right: $block-padding*2 + 2px;
+
+		@include break-medium() {
+			padding-left: $block-padding*2;
+			padding-right: $block-padding*2;
+		}
+	}
+}
+
+
+/**
+ * Section Full Alignment
+ */
+.wp-block[data-type="core/#{$block-name}"][data-align="full"] {
+
+	.editor-inner-blocks {
+		padding-left: 46px;
```

Comment:
For the final PR, it'd be excellent to spell out what these pixel measurements consist of. (I'm not totally sure myself 😄)